plugins/win-mf/win-mf.cpp

```diff
@@ -0,0 +1,1736 @@
+		char description[256];
+		winrt::check_hresult(candidateAdapter->GetProperty(
+			DXCoreAdapterProperty::DriverDescription,
+			&description));
+
+		char npu[256] = "Intel(R) AI Boost";
+		if (strcmp(description, npu) == 0)
+			npuDetected = true;
+	}
+
+	return npuDetected;
+}
+
+static obs_properties_t *GetMediaFoundationSourceProperties(void *obj)
+{
+	MediaFoundationSourceInput *input =
+		reinterpret_cast<MediaFoundationSourceInput *>(obj);
+	obs_properties_t *ppts = obs_properties_create();
+	PropertiesData *data = new PropertiesData;
+
+	data->input = input;
+
+	obs_properties_set_param(ppts, data, PropertiesDataDestroy);
+
+	obs_property_t *p = obs_properties_add_list(ppts, VIDEO_DEVICE_ID,
+						    TEXT_DEVICE,
+						    OBS_COMBO_TYPE_LIST,
+						    OBS_COMBO_FORMAT_STRING);
+
+	obs_property_set_modified_callback(p, DeviceSelectionChanged);
+
+	DeviceEnumerator mfenum;
+	mfenum.Enumerate(data->devices);
+
+	for (const MediaFoundationVideoDevice &device : data->devices)
+		AddDevice(p, device);
+
+	const char *activateText = TEXT_ACTIVATE;
+	if (input) {
+		if (input->_active)
+			activateText = TEXT_DEACTIVATE;
+	}
+
+	obs_properties_add_button(ppts, "activate", activateText,
+				  ActivateClicked);
+
+	obs_properties_add_bool(ppts, DEACTIVATE_WNS, TEXT_DWNS);
+
+	/* ------------------------------------- */
+	/* Intel NPU AI effect settings */
+
+	if (NPUDetection()) {
+
+		p = obs_properties_add_list(ppts, INTELNPU_BLUR_TYPE,
+					    TEXT_INTELNPU_BLUR_TYPE,
+					    OBS_COMBO_TYPE_LIST,
+					    OBS_COMBO_FORMAT_INT);
+
+		obs_property_list_add_int(p, TEXT_INTELNPU_BLUR_NONE,
+					  NpuBlurType_None);
+		obs_property_list_add_int(p, TEXT_INTELNPU_BLUR_STANDARD,
+					  NpuBlurType_Standard);
+		obs_property_list_add_int(p, TEXT_INTELNPU_BLUR_PORTRAIT,
+					  NpuBlurType_Portrait);
+
+		obs_properties_add_bool(ppts, INTELNPU_BACKGROUND_REMOVAL,
+					TEXT_INTELNPU_BACKGROUND_REMOVAL);
+		obs_properties_add_bool(ppts, INTELNPU_AUTO_FRAMING,
+					TEXT_INTELNPU_AUTO_FRAMING);
+		obs_properties_add_bool(ppts, INTELNPU_EYEGAZE_CORRECTION,
+					TEXT_INTELNPU_EYEGAZE_CORRECTION);
+	}
+
+	/* ------------------------------------- */
+	/* video settings */
+
+	p = obs_properties_add_list(ppts, RES_TYPE, TEXT_RES_FPS_TYPE,
+				    OBS_COMBO_TYPE_LIST, OBS_COMBO_FORMAT_INT);
+
+	obs_property_set_modified_callback(p, ResTypeChanged);
+
+	obs_property_list_add_int(p, TEXT_PREFERRED_RES, ResTypePreferred);
+	obs_property_list_add_int(p, TEXT_CUSTOM_RES, ResTypeCustom);
+
+	p = obs_properties_add_list(ppts, RESOLUTION, TEXT_RESOLUTION,
+				    OBS_COMBO_TYPE_EDITABLE,
+				    OBS_COMBO_FORMAT_STRING);
+
+	obs_property_set_modified_callback(p, DeviceResolutionChanged);
+
+	p = obs_properties_add_list(ppts, FRAME_INTERVAL, "FPS",
+				    OBS_COMBO_TYPE_LIST, OBS_COMBO_FORMAT_INT);
+
+	obs_property_set_modified_callback(p, DeviceIntervalChanged);
+
+	obs_property_set_long_description(p,
+					  obs_module_text("Buffering.ToolTip"));
```

Comment:
This configures the "Buffering.ToolTip" text onto the "FPS" dropdown. If there is no plan to keep Buffering, I recommend going through and deleting related buffering code, otherwise re-introduce the Buffering toggle and assign this tooltip to it.